src/Microsoft.Diagnostics.ExtensionCommands/LiveObjectService.cs

```diff
@@ -0,0 +1,93 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using Microsoft.Diagnostics.DebugServices;
+using Microsoft.Diagnostics.Runtime;
+
+namespace Microsoft.Diagnostics.ExtensionCommands
+{
+    [ServiceExport(Scope = ServiceScope.Runtime)]
+    public class LiveObjectService
+    {
+        public int UpdateSeconds { get; set; } = 15;
+
+        private ObjectSet _liveObjs;
```

Comment:
Nit: I usually put private fields first in a class then protected and then public fields/properties.